lib/galaxy/managers/configuration.py

```diff
@@ -40,6 +40,7 @@ def _required_attribute(item, key, **context):
             # TODO: this is available from user data, remove
             'is_admin_user'                     : lambda *a, **c: False,
             'brand'                             : _required_attribute,
+            'brand_enabled'                     : _defaults_to(True),
```

Comment:
you can use `_required_attribute` - that way it will default to the default value specified in the schema. The only reason for using `_defaults_to` in that file is to address special cases - when the schema's default is different or when the option  is not defined in the schema. (maybe `_required_attribute` was not the best name choice, it's definitely not intuitive...)

Reply:
Cool, thank you!